test/TileCacheTests.cpp

```diff
@@ -19,6 +19,9 @@
 
 #include <cppunit/extensions/HelperMacros.h>
 
+#include <sstream>
```

Comment:
Formatting changes to your existing code should be squashed into the existing commit. But then I guess for back-porting it's good to have cosmetic cleanup separated I suppose.

Reply:
It's formatting changes for new code so you're right it's probably better to squash it.